src/index.js

```diff
@@ -1,79 +1,89 @@
-import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import styled from 'styled-components';
 
-import Table from './Table';
-import Row from './Row';
-import Cell from './Cell';
+const Table = styled.div`
+  white-space: nowrap;
+  margin-top: 2rem;
+  border: 2px solid #e5e5e5;
+  display: table;
+  box-sizing: border-box;
+`;
 
-var index = 0;
+Table.displayName = 'Table';
 
-/**
- * StickyTable Component
- * Responsive, dynamically sized fixed headers and columns for tables
- * ------------------------------------------------------------------
- * Intentionally not setting state because we don't want to require
- * a full re-render every time the user scrolls or changes the
- * width of a cell.
- */
-class StickyTable extends PureComponent {
-  static propTypes = {
-    stickyHeaderCount: PropTypes.number,
-    stickyColumnCount: PropTypes.number,
-    onScroll: PropTypes.func
-  };
+const Row = styled.div`
+  display: table-row;
 
-  static defaultProps = {
-    stickyHeaderCount: 1,
-    stickyColumnCount: 1
-  };
+  & ${Cell}:first-child {
+    border-right: 2px solid #e5e5e5;
+  }
+`;
 
-  constructor(props) {
-    super(props);
+Row.displayName = 'Row';
 
-    this.index = index = index + 1;
-  }
+const Cell = styled.div`
```

Comment:
@horacioh I think these cells should still have the classNames they had before so people who do use css can easily select them. I think the tests should include those selectors as well, as they did before to be clear about what is being selected rather than just 'span's and 'div's.

Reply:
yeah this is something I was thinking before, we can leave the classes, but with no extra css for them, just for users to override it.

another option i thought was to have this refactor as a breaking change. but I dunno about how people will react to this...

I'm OK with both options!

about the tests, I think targeting the actual components is clear enough. what do you think is different from this compared to css classes?

Reply:
Let's keep the classes. Will comment inline in test file to show

Reply:
done!